src/ape/utils.py

```diff
@@ -216,6 +296,16 @@ def extract_nested_value(root: Mapping, *args: str) -> Optional[Dict]:
 
 
 def stream_response(download_url: str, progress_bar_description: str = "Downloading") -> bytes:
+    """
+    To prvoide a visual represenation of the downloads.
+
+    Args:
+        download_url (str): String to get files to download.
+        progress_bar_description (str): Downloading word.
+
+    Returns:
+        bytes (bytes): Content in bytes to show the progress.
```

Comment:
```suggestion
        bytes: Content in bytes to show the progress.
```

^ The first of the `Returns:` block _is_ the type.
It is different than the `Args:` section